### PR TITLE
style and other fixes

### PR DESCRIPTION
**Pull request by nickcarr51**
- all sorts of styling fixes for consistency (input styling, buttons in places we missed, alerts replaced with toasts) and just general things to make it look a little more sleek 
- added error handling if admin tries to add movie to the database that's already in the database
- added error handling so there's a toast notification for wrong password
- 'username already in use' now displays in red on the signup page
- started deleting console.logs/commented out code
- removed add to wishlist (maybe someday...)
- admin previous orders now sorted by date order
- checkout re-directs to browse instead of home (get them buying more movies!)

### CODE REVIEW

File: client/components/browse.jsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Browse extends Component {
   }
 
   async componentDidMount() {
+    window.scrollTo(0, 0);
```
Comment (chadalad) [inline client/components/browse.jsx:23]: good addition

File: client/components/browse.jsx [hunk 2]
```diff
@@ -119,103 +108,3 @@ const mapStatetoProps = (state) => ({
 const mapDispatchToProps = { getMovies };
 
 export default connect(mapStatetoProps, mapDispatchToProps)(Browse);
-
```
Comment (chadalad) [inline client/components/browse.jsx:122]: heyooo comments deleted!

File: server/api/routers/movies.js [hunk 3]
```diff
@@ -25,38 +25,46 @@ movieRouter.post('/imdbsearch', async (req, res) => {
...
-    year: movie.year,
-    price: 0.99,
-    stock: 50,
-  });
+  const checkMovie = await Movie.findAll({ where: { id } });
 
-  if (newMovie) {
-    res.send(newMovie);
+  if (checkMovie) {
+    res.sendStatus(204);
```
Comment (chadalad) [inline server/api/routers/movies.js:31]: 204 is "no content" is that what we want to send?

Comment (nickcarr51) [reply server/api/routers/movies.js:31]: anything in the 400's seems to bring the page to a halt, this allows us to throw up a toast notification

File: server/api/routers/users.js [hunk 4]
```diff
@@ -51,7 +48,7 @@ userRouter.post('/login', async (req, res) => {
         }
         await res.status(200).send(user);
       } else {
-        res.sendStatus(400);
+        res.sendStatus(204);
```
Comment (chadalad) [inline server/api/routers/users.js:51]: 200 statuses are successes 
400 statuses are client errors - I think we would want a 404 not found?

File: client/components/moviePage.jsx [hunk 5]
```diff
@@ -136,12 +126,14 @@ class MoviePage extends Component {
                         ? (
                           <p className="title is-6" style={{ marginTop: '20px' }}>
                             Directors:
+                            {' '}
```
Comment (chadalad) [inline client/components/moviePage.jsx:129]: for these, could "\n" be used in line?

**Pull request merged**